xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-api/src/main/java/org/xwiki/mentions/MentionNotificationService.java

```diff
@@ -35,13 +36,14 @@
 {
     /**
      * Send a notification on behalf of the author, informing the mentioned user that he/she is mentioned on the a page.
-     *
-     * @param authorReference the reference of the author of the mention.
+     *  @param authorReference the reference of the author of the mention.
      * @param documentReference the document in which the mention has been done.
      * @param mentionedIdentity the identity of the mentioned user.
      * @param location The location of the mention.
      * @param anchorId The anchor link to use.
+     * @param xdom the content xdom
      */
-    void sendNotif(DocumentReference authorReference, DocumentReference documentReference,
-        DocumentReference mentionedIdentity, MentionLocation location, String anchorId);
+    void sendNotification(DocumentReference authorReference, DocumentReference documentReference,
```

Comment:
instead of adding more and more arguments there, an object should probably be sent.